trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -449,7 +449,7 @@ const confirmTfmFacility = async (req, res) => {
       api
         .getParty(urn, userToken)
         // Non-existent party urn
-        .then((company) => (!company || !company.data ? Promise.resolve(false) : Promise.resolve(true))));
+        .then((company) => (!company || !company.data || company.status !== 200 ? Promise.resolve(false) : Promise.resolve(true))));
```

Comment:
It should be `Promise.reject()` instead of `Promise.resolve(false)`, thoughts?

Reply:
Optional chaining would be good have.

Reply:
Will add the chaining, I only didn't change to promise.reject as it's existing behaviour I didn't want to impact. Happy to move to promise.reject

Reply:
In that case if the calling functions rely on `false` respone then happy to leave it as `Promie.resolve(false)`. Thanks